src/analysis/calculus/times_cont_diff.lean

```diff
@@ -1880,10 +1880,10 @@ begin
     exact (h i).zero_eq x hx },
   { intros m hm x hx,
     have := has_fderiv_within_at_pi.2 (λ i, (h i).fderiv_within m hm x hx),
-    convert (L m).has_fderiv_at.comp_has_fderiv_within_at x this },
+    exact (L m).has_fderiv_at.comp_has_fderiv_within_at x this },
```

Comment:
These may have been intentional, as `convert` is sometimes faster than `exact`. Perhaps worth checking when this was last changed?

Reply:
Sorry, that was my doing. In the original version of this PR (#7521) there was a problem with this proof, and when I was trying to fix it I realized that the `convert`s could be `exact`s. This proof was quite slow and I thought doing this would help speed it up, of course if that is not the case I will revert it.

Reply:
It looks like Yury wrote those lines in #6674, but I don't see anything that indicates that `convert` was used for a specific reason.

Reply:
I timed the code roughly, and it does seem like `covert` speeds things up a bit, so I changed `exact` back to `convert`

Reply:
It seems `convert` is a fraction of a second faster here, but not enough to care. As you were. :-)

Reply:
Cheers, thanks.